src/main/java/com/kryptnsotic/rhizome/converters/RhizomeConverter.java

```diff
@@ -0,0 +1,80 @@
+package com.kryptnsotic.rhizome.converters;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.lang.reflect.Type;
+
+import org.apache.commons.io.IOUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import retrofit.converter.ConversionException;
+import retrofit.converter.Converter;
+import retrofit.mime.TypedByteArray;
+import retrofit.mime.TypedInput;
+import retrofit.mime.TypedOutput;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JavaType;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.geekbeast.rhizome.registries.ObjectMapperRegistry;
+
+/**
+ * A {@link Converter} that handles both json and byte arrays. Based off work by Kai Waldron
+ * (kaiwaldron@gmail.com)
+ * 
+ * @author Matthew Tamayo-Rios &lt;matthew@kryptnostic.com&gt;
+ *
+ */
+public class RhizomeConverter implements Converter {
```

Comment:
It's used a lot in the kryptnosticRestAdapter: https://github.com/kryptnostic/iris/blob/develop/src/main/java/com/kryptnostic/api/v1/client/KryptnosticRestAdapter.java